build.gradle

```diff
@@ -145,7 +141,7 @@ project ('hdfs') {
                 "io.pravega:pravega-segmentstore-contracts:${pravegaVersion}",
                 "io.pravega:pravega-segmentstore-storage:${pravegaVersion}"
         compile  "io.pravega:pravega-test-testcommon:${pravegaVersion}"
-        compile files("$rootDir/libs/pravega-segmentstore-storage-0.9.0-2641.ea37ccadc-issue-5124-use-maven-plugin-tests.jar")
```

Comment:
since we publish test jars from Pravega Core now please remove all jars from `libs` folder. We no longer need them.